### PR TITLE
lighttpd: bump version

### PR DESCRIPTION
**Pull request by kenmays**

### CODE REVIEW

**Comment by extrowerk** [comment]
Builds/packages fine on 64 bit.

**Comment by Begasus** [comment]
Builds/packages OK on 32 bit also, but running the tests crashes ...

File: www-servers/lighttpd/lighttpd-1.4.55.recipe [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+	create-lighttpd-directories.sh
+	index.template
+	geoip.shtml
+	quickstart.txt
+	quickstart-geoip.txt
+	"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64 ?arm ?ppc"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
```
Comment (Begasus) [inline www-servers/lighttpd/lighttpd-1.4.55.recipe:25]: Change:
```diff
-SECONDARY_ARCHITECTURES="x86"
+SECONDARY_ARCHITECTURES="?x86"
```

**Comment by kenmays** [comment]
Fixed. Reverted to v1.4.53.

**Comment by kenmays** [comment]
Successfully ran 58,269 Webkit 611.1.1 tests with lighttpd 1.4.53 enabled - with no crashes. Passed all 10 unit tests. Good enough.

**Comment by pulkomandy** [comment]
Thanks for checking!

**Pull request merged**